gensim/models/keyedvectors.py

```diff
@@ -55,8 +55,12 @@
 
 """
 from __future__ import division  # py3 "true division"
+from gensim.models import TfidfModel
+from gensim.corpora import Dictionary
+from scipy.sparse.linalg import svds
```

Comment:
We have our own decomposition, look at `gensim.models.lsimodel`

Reply:
I just need [this](https://github.com/RaRe-Technologies/gensim/blob/49e6abdde6bca55b7a5b263c01ecdc94a75fb1ab/gensim/models/lsimodel.py#L293) step not sure if it is useful.